src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravVurderingQueries.kt

```diff
@@ -183,6 +184,25 @@ fun DatabaseInterface.getAktivitetskrav(
     }
 }
 
+const val queryGetOutdatedAktivitetskrav = """
+    SELECT * 
+    FROM AKTIVITETSKRAV
+    WHERE stoppunkt_at > ?
+    AND stoppunkt_at < ?
+    AND status = 'NY'
+    AND personident NOT IN (SELECT personident FROM AKTIVITETSKRAV WHERE stoppunkt_at >= ?);
```

Comment:
Er dette tilfeller hvor det finnes et gammelt, ubehandlet aktivitetskrav, i tillegg til et nytt? Burde vi ikke egentlig lukke de også? Kanskje i det vi lager et nytt?

Reply:
> Er dette tilfeller hvor det finnes et nytt aktivitetskrav i tillegg til et gammelt, ubehandlet for en gitt innbygger? Burde vi ikke egentlig lukke de også? Kanskje i det vi lager et nytt?

Jo, vi lukker gammelt, ubehandlet aktivitetskrav når vi genererer et nytt. Så dette blir egentlig bare for å fange opp tilfeller hvor en innbygger har et gammelt, ubehandlet aktivitetskrav og så vært frisk en stund før veileder har gått inn og vurdert aktivitetskrav uten at vi har generert et nytt på forhånd.
Får bare én ekstra rad i resultatet hvis jeg fjerner betingelsen, men tenker det greit å ta den med siden vi ikke ønsker å lukke denne (den ligger jo som vurdert og da uten oppgave i oversikten).

Reply:
Kanskje lurt å legge på en LIMIT her hvis det kan bli et stort resultatsett (for å unngå OutOfMemory-feil).

Reply:
Kunne vært lurt, men det er ca 200 i prod-databasen så tror ikke det trengs her :)